tests/integration/docusaurus/connecting_to_your_data/cloud/gcs/pandas/configured_python_example.py

```diff
@@ -17,6 +17,9 @@
             "class_name": "ConfiguredAssetGCSDataConnector",
             "bucket_or_name": "<YOUR_GCS_BUCKET_HERE>",
             "prefix": "<BUCKET_PATH_TO_DATA>",
+            "gcs_options": {
+                "delimiter": "/",
+            },
```

Comment:
How about just passing the delimiter directly via the delimiter param?

```suggestion
            "delimiter": "/",
```